### ISSUE TITLE
CLI, set and exit?

### ISSUE THREAD

**Issue by bitwombat** [MohamadSaada/LogiGSK#13]
Hi,

Thanks for putting this together!

However, it does not suit my use case - I do not want yet another daemon running and taking up memory.  I have a simple G610 (monochrome) and just want to set LEDs upon X.org starting... like xset does for NumLock, CapsLock and ScrollLock.

Any plans to support that use case?

**Comment by MohamadSaada** [comment]
@bitwombat this is possible to do but a problem that you might face is that whenever your computer suspends the keyboard will reset to its default mode.

**Comment by bitwombat** [comment]
Ah, interesting.  Aren't there scripts in /etc/ that get run on resume?

**Comment by MohamadSaada** [comment]
@bitwombat you're correct, the other main problem is that there will have to be a major rewrite of the code to apply this change, as everything is written with a daemon in mind. I will see what I can do.
